packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -393,7 +393,7 @@ function WorklistDataGrid(props) {
 
   const handleColumnSelect = (columnId, cellCoordinates) => {
     handleCellSelectionChange(null, null, cellCoordinates);
-    if (onColumnSelect && !(hasSelectableRows && cellCoordinates.col === 0)) {
+    if (onColumnSelect) {
```

Comment:
Should we have WDIO/Jest tests for this change?

Reply:
I don't think I can do a WDIO test because we are just executing a callback function.  Can I create a jest test for worklist data grid even though the component being clicked is a couple levels deep?